src/components/NavBar.tsx

```diff
@@ -0,0 +1,42 @@
+"use client";
```

Comment:
curious if we can SSR this and pass it through sockets. I haven't figured it out yet